src/Sulu/Bundle/ContactBundle/Resources/translations/sulu/backend.en.xlf

```diff
@@ -642,6 +642,22 @@
                 <source>labels.success.user.activated</source>
                 <target>User enable</target>
             </trans-unit>
+            <trans-unit id="235b14f26g712345e44gs2e9335a8d63" resname="public.socialMediaProfile">
+                <source>public.socialMediaProfile</source>
+                <target>Social media profile</target>
+            </trans-unit>
+            <trans-unit id="235b14f26g712345e44gs2e9335a8d73" resname="social_media_profile.facebook">
+                <source>social_media_profile.facebook</source>
```

Comment:
I think I would replace all of these translations with a simple string... The names never change, and it makes adding a new one harder... But let me discuss that with @chirimoya first.